apps/dashboard/test/system/batch_connect_test.rb

```diff
@@ -1444,8 +1444,8 @@ def make_bc_app(dir, form)
   def get_favorites
     # For debugging flaky tests
     favorites = all('#favorites li', wait: 30)
-    puts "FAVORITES: "
-    puts favorites.map{|i| i['innerHTML']}.join('')
+    # puts "FAVORITES: "
+    # puts favorites.map{|i| i['innerHTML']}.join('')
```

Comment:
This change isn't a part of this really - I just wanted to get rid of this stdout in tests.